src/ja/server/main.py

```diff
@@ -30,6 +33,7 @@ def _init_algorithm() -> SchedulingAlgorithm:
     @staticmethod
     def _read_config() -> ServerConfig:
         with open('/etc/jobadder/server.conf') as f:
+            logger.info("read config")
```

Comment:
I think a more helpful message would include the name of the config file being read.